lib/jira/client/index.js

```diff
@@ -144,6 +148,36 @@ module.exports = async (jiraHost, gitHubInstallationId, logger) => {
   return client
 }
 
+/**
+ * Splits commits in data payload into chunks of 400 and makes separate requests
+ * to avoid Jira API limit
+ */
+const batchedCommitRequest = async (data, options, instance, installationId) => {
```

Comment:
```suggestion
const batchedBulkUpdate = async (data, options, instance, installationId) => {
```

After reading this code, I can't help but think the original name is under selling the utility of this method.